### PR TITLE
maestro-cloud-upload 1.2.0

### PR DESCRIPTION
**Pull request by ArthurSav**
TagCheck

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [X] __I will not move an already shared step version's tag to another commit__
- [X] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [X] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [X] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [X] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

**Comment by ArthurSav** [comment]
Hey @tothszabi, would really appreciate a review 🙏
It's a minor field addition

**Comment by axelniklasson** [comment]
@tothszabi Just checking in here -- is there anything we can do to help speed up the review process? Thanks!

File: steps/maestro-cloud-upload/1.2.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+    is_required: false
+    summary: 'Set the Android api level the devices should run on (Default: 30)'
+    title: Android api level
+- ios_version: ""
+  opts:
+    description: ""
+    is_expand: true
+    is_required: false
+    summary: 'Set the iOS version the devices should run on (Default: 15)'
+    title: Android api level
```
Comment (tothszabi) [inline steps/maestro-cloud-upload/1.2.0/step.yml:108]: The `ios_version` input has the same as the `android_api_level`. Is this a copy paste error?

Comment (axelniklasson) [reply steps/maestro-cloud-upload/1.2.0/step.yml:108]: Yes that is correct, thanks for spotting this -- let me update

Comment (axelniklasson) [reply steps/maestro-cloud-upload/1.2.0/step.yml]: @tothszabi fixed in https://github.com/bitrise-io/bitrise-steplib/pull/3868/commits/108ad7650621e719215ab8962175be8b2c90a435

**Review by ofalvai — APPROVED** [review]

**Pull request merged**